### ISSUE TITLE
[BUG] IOS Content.json fails to build on ios

### ISSUE THREAD

**Issue by huideaki** [mrrhak/icons_launcher#52]
### :information_source: Info
Version: `v3.0.0-beta.2`

### :speech_balloon: Description

Using the current 3.0.0-beta (1 or 2) causes the Contents.json to be different than previous versions.

the info of Contents.json on 2.1.7
```
  "info": {
    "author": "icons_launcher",
    "version": 1
  }
```

the info of Contents.json on 3.0.0-beta2
```
  "info": {
    "author": "icons_launcher",
    "version": "3.0.0-beta.2",
    "github": "https://github.com/mrrhak/icons_launcher"
  }
```

this causes a compilation failure in flutter due to flutter ios build casting the version as an int? it might be an issue on thier side but currently this stops the library of being used. ref : https://github.com/flutter/flutter/issues/149106

**Comment by mrrhak** [comment]
Thank you for reporting I will check and update soon.

**Comment by mrrhak** [comment]
Hi @huideaki, I have fixed this on [v3.0.0-beta.3](https://pub.dev/packages/icons_launcher/versions/3.0.0-beta.3). Thanks
